oclc.py

```diff
@@ -750,28 +750,34 @@ def run(self):
                     output.write("\n")
 
 
-class LinkedDataCoverageProvider(CoverageProvider):
+class LinkedDataCoverageProvider(IdentifierCoverageProvider):
```

Comment:
Similar story here as with Content Cafe. We don't care which collection or catalog these identifiers belong to, and it doesn't help us get the data (e.g. by providing credentials) since the service is public.